main.go

```diff
@@ -101,6 +101,11 @@ func NewApp() *cli.App {
 			Name:  "up",
 			Usage: "Create database (if necessary) and migrate to the latest version",
 			Flags: []cli.Flag{
+				&cli.BoolFlag{
+					Name:    "strict",
+					EnvVars: []string{"DBMATE_STRICT"},
+					Usage:   "apply migrations strictly in numerical order",
```

Comment:
I think we need a better description here. I put one suggestion below. The flag is not "apply migrations strictly in numerical order", because we always do that. The point is that we fail if existing applied migrations have been applied out of order, if I understand correctly?

```suggestion
					Usage:   "fail if existing migrations have been applied out of order",
```

Reply:
True, the description should be more precise. The current PR only applies for new/pending migrations. It fails if a new migration has a smaller version number than the already applied migration with the highest version number.